### ISSUE TITLE
Connect Aasonida's desktop light to Google Home

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#593]
# Issue
Aasonida's light https://github.com/130s/hut_10sqft/issues/575#issuecomment-909829700 seems to be compatible with Tuya's apps. The number of apps to operate daily would add more.

# CoS
Ideally not add a new app to operate daily.

**Comment by 130s** [comment]
Tuya app  is compatible with Google Home so going on that route.

Tuya's app lineup is confusing. Following Aasonida's user manual, I have "Smart Life" app installed, which is the [same app as Tuya Smart app (support.tuya.com)](https://support.tuya.com/en/help/_detail/K9hrdz10p224e). So first I'll follow the manual and connect the device to Smart Life app, but later I'll try connecting the device to Tuya Smart app instead of Smart Life.

**Comment by 130s** [comment]
1. [x] Connect the device to Smart Life app. Operate from app.
1. [x] Connect the device to Tuya Smart app. Operate from app.
1. [x] Device control works from Google Home, whatever features available.
1. [x] Uninstall Smart Life app. Device control still works from Tuya Smart app.

# Rant
Connecting the device to the app from either app took dozens of trials...Neither instruction in the manual nor the app make it clear enough.

IIUC, the order of the user input the app asks: 1) Asks LAN wifi creds that the device should connect to. 2) Asks the user to set the device in order to reset and prepare for the next step (which enables the device's local wifi). 3) Asks the user to connect the app to the local wifi. With a proper connection established, app will automatically handles the rest.

However I needed dozens of trials to move from step 2 to 3 above. I still don't know what I did finally got it move forward. Just a guess, one of the keys might be to unplug and replug the power cable of the device at the beginning of step 2.

**Comment by 130s** [comment]
Redid this from Tuya Smart app again and went fine just a single attempt.

**Comment by 130s** [comment]
Can't find where the manual is anymore...

Device went offline for unknown timing and cause. I managed to reconnect it.
1. Remove from Tuya android app. (I chose "delete" but didn't choose "delete and delete data).
1. On The same Tuya app, start connect new.
1. Lighting -> Desktop light w/Wifi.
1. Next window suggested startup procedure by pressing on-off-on-off-on, which didn't do anything on my device. So instead I did, turn off power, press power button for 10+ secs. This lit the burger/wireless network light blink, which is probably what the instruction needs.
1. Back to app, click next. Ignore light condition that it asks.
1. Confirm 2.4 Ghz wifi creds.
1. App should start finding the device and it finishes. 
   - In my case it took 2+ mins to finish so be patient.
